packages/integration-tests/test/childchain-exit-test.js

```diff
@@ -0,0 +1,120 @@
+    let balance = await helper.waitForBalance(childChain, srcAccount.address, TRANSFER_AMOUNT)
+    // Check src account balance is correct
+    assert.equal(balance.length, 1)
+    assert.equal(balance[0].amount.toString(), TRANSFER_AMOUNT)
+    console.log(`Deposited ${TRANSFER_AMOUNT} into source account ${srcAccount.address}`)
+
+    // Check utxos on the child chain
+    const utxos = await childChain.getUtxos(srcAccount.address)
+    assert.equal(utxos.length, 1)
+    assert.hasAllKeys(utxos[0], ['txindex', 'txbytes', 'oindex', 'currency', 'blknum', 'amount'])
+    assert.equal(utxos[0].amount.toString(), TRANSFER_AMOUNT)
+    assert.equal(utxos[0].currency, ETH_CURRENCY)
+
+    // Convert it to a Number from BigNumber
+    utxos[0].amount = Number(utxos[0].amount)
+    const txBody = {
+      inputs: [utxos[0]],
+      outputs: [{
+        owner: destAccount.address,
+        amount: Number(TRANSFER_AMOUNT)
+      }]
+    }
+
+    console.log(`Spending utxo ${JSON.stringify(utxos[0])}`)
+
+    // Create the unsigned transaction
+    const unsignedTx = await childChain.createTransaction(txBody)
+    // Sign it
+    const signatures = await childChain.signTransaction(unsignedTx, [srcAccount.privateKey])
+    assert.equal(signatures.length, 2)
+    // Build the signed transaction
+    const signedTx = await childChain.buildSignedTransaction(unsignedTx, signatures)
+    // Submit the signed transaction to the childchain
+    const result = await childChain.submitTransaction(signedTx)
+    console.log(`Submitted transaction: ${JSON.stringify(result)}`)
+
+    // destAccount balance should be TRANSFER_AMOUNT
+    balance = await helper.waitForBalance(childChain, destAccount.address, TRANSFER_AMOUNT)
+    assert.equal(balance.length, 1)
+    assert.equal(balance[0].amount.toString(), TRANSFER_AMOUNT)
+    console.log(`Transferred ${TRANSFER_AMOUNT} from source account ${srcAccount.address} to dest account ${destAccount.address}`)
+
+    // srcAccount balance should be zero
+    balance = await childChain.getBalance(srcAccount.address)
+    assert.equal(balance.length, 0)
+
+    // destAccount wants to exit
+    const destUtxos = await childChain.getUtxos(destAccount.address)
+    assert.equal(destUtxos.length, 1)
+    assert.equal(destUtxos[0].amount.toString(), TRANSFER_AMOUNT)
+
+    // We're exiting with the last (and only) utxo of destAccount
+    const utxoToExit = destUtxos[0]
+    const exitData = await childChain.getExitData(utxoToExit)
+
+    let receipt = await rootChain.startExit(
+      destAccount.address,
+      exitData.utxo_pos.toString(),
+      exitData.txbytes,
+      exitData.proof,
+      exitData.sigs,
+      destAccount.privateKey
+    )
+    console.log(`Destination account called RootChain.startExit(): txhash = ${receipt.transactionHash}`)
+
+    // Wait for challenge period
+    console.log(`Waiting for challenge period... ${CHALLENGE_PERIOD}ms`)
+    await helper.sleep(CHALLENGE_PERIOD)
```

Comment:
so without extra configuration, the unit test would take about 2 weeks to run LOL